core/src/main/java/tech/pegasys/eth2signer/core/multikey/metadata/FileRawSigningMetadata.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.eth2signer.core.multikey.metadata;
+
+import tech.pegasys.eth2signer.core.signing.ArtifactSigner;
+import tech.pegasys.eth2signer.crypto.KeyPair;
+import tech.pegasys.eth2signer.crypto.SecretKey;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.tuweni.bytes.Bytes;
+
+public class FileRawSigningMetadata implements SigningMetadata {
+
+  private final String privateKey;
+
+  @JsonCreator
+  public FileRawSigningMetadata(
+      @JsonProperty(value = "privateKey", required = true) final String privateKey) {
+    this.privateKey = privateKey;
+  }
+
+  @JsonProperty(value = "privateKey")
+  public String getPrivateKey() {
+    return privateKey;
+  }
+
+  @Override
+  public ArtifactSigner createSigner() {
+    final Bytes privateKey1 = Bytes.fromHexString(getPrivateKey());
```

Comment:
maybe privateKeyBytes rather than privateKey1?

If this fails, because the string is invalid - it won't be found at parse, but at usage - is that bad?

Reply:
whoops, I that was meant to be privateKeyBytes

Reply:
Including the conversion as part of the constructor now so if it's invalid we throw a SigningMetadataException